nilearn/glm/first_level/first_level.py

```diff
@@ -909,6 +873,90 @@ def _get_voxelwise_model_attribute(self, attribute, result_as_time_series):
 
         return output
 
+    def _prepare_mask(self, run_img):
+        """Set up the masker.
+
+        Parameters
+        ----------
+        run_img : Niimg-like object or SurfaceImage object
+            Used for setting up the masker object.
+        """
+        # Local import to prevent circular imports
+        from nilearn.maskers import NiftiMasker
+
+        # Learn the mask
+        if self.mask_img is False:
```

Comment:
or None ?

Reply:
@bthirion `mask_img=None` is directly passed to initialization of either `SurfaceMasker` in L906 or `NiftiMasker` in L918. Note that in the case of `NiftiMasker` the behavior of `None` and `False` are different because `None` leads to automatic computation of a mask and `False` leads to no masking. Since `SurfaceMasker` doesn't implement automatic mask computation the behavior is the same.